lib/config/main_test.go

```diff
@@ -118,6 +121,11 @@ func checkConfigExpectation(t *testing.T, expectedConfig *oplogtoredisConfigurat
 			expectedConfig.RedisURL, RedisURL())
 	}
 
+	if expectedConfig.RedisTLS != RedisTLS() {
+		t.Errorf("Incorrect Redis TLS. Got \"%t\", Expected \"%t\"",
```

Comment:
Did you mean `%T` in both places in `t.Errorf()`? Otherwise that's just going to print `true` always, these are non-nil structs

Reply:
I believe 
%T	a Go-syntax representation of the type of the value
and
%t	the word true or false.
In this case, since expectedConfig.RedisTLS, RedisTLS() are both returning booleans, I thought %t would be appropriate. I guess I could just do %v in case RedisTLS is nill.

Reply:
No ,I misread this; I thought you were printing out a struct, not a bool. So forget it!